eslint.config.mjs

```diff
@@ -0,0 +1,79 @@
+import globals from "globals";
+import parser from "astro-eslint-parser";
+import tsParser from "@typescript-eslint/parser";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+    allConfig: js.configs.all
+});
+
+export default [{
+    ignores: [
+        "**/node_modules/**",
+        "**/dist/**",
+        "**/package-lock.json",
+        "**/*.md",
+        "**/*.min.js",
+        "content/**/*",
+        "public/content/**/*",
+        ".git/**",
+        ".vscode/**",
+        "coverage/**"
```

Comment:
Why are we covering these three lines? `.git` I could see an argument for, but `.vscode` and `coverage` likely shouldn't be there.